Analysis/Core/include/AnalysisCore/HFSelectorCuts.h

```diff
@@ -331,6 +332,52 @@ static const std::vector<std::string> pTBinLabels = {
 static const std::vector<std::string> cutVarLabels = {"m", "pT p", "pT K", "pT Pi", "DCA", "vertex sigma", "decay length", "cos pointing angle"};
 } // namespace hf_cuts_xic_topkpi
 
+namespace hf_cuts_xicc_topkipipi
+{
+static constexpr int npTBins = 9;
+static constexpr int nCutVars = 6;
+// default values for the pT bin edges (can be used to configure histogram axis)
+// offset by 1 from the bin numbers in cuts array
+constexpr double pTBins[npTBins + 1] = {
+  0,
+  0.5,
+  1.0,
+  2.0,
+  3.0,
+  4.0,
+  5.0,
+  7.0,
+  10.0,
+  15.0,
+};
+auto pTBins_v = std::vector<double>{pTBins, pTBins + npTBins + 1};
+
+// default values for the cuts
+//                                            m    CPA  d0Xic  d0Pi  pTXic pTPi
+constexpr double cuts[npTBins][nCutVars] = {{0.5, 0.90, 0.001, 0.001, 2.0, 0.3},  /* 0<pt<0.5 */
+                                            {0.5, 0.90, 0.001, 0.001, 2.0, 0.3},  /* 0.5<pt<1 */
+                                            {0.5, 0.90, 0.001, 0.001, 2.0, 0.3},  /* 1<pt<2   */
```

Comment:
Please adjust the spacing in comments for better readability.

Reply:
done